sys/fido2/doc.txt

```diff
@@ -102,6 +106,9 @@
  *
  * Abstraction for flash operations. Uses the RIOT [Flashpage MTD driver](http://api.riot-os.org/group__drivers__mtd__flashpage.html).
  *
+ * Flash memory is reserved at build time using the `FLASH_WRITABLE_INIT` macro. The amount of flash
+ * reserved can be configured as `FIDO2_CTAP_NUM_FLASHPAGES` in KConfig.
```

Comment:
Similarly to the Kconfig help, it would be nice to explain here that 1. page is always state, and subsequent pages are used to store (multiple) rks.